src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -0,0 +1,104 @@
+import sys
+
+import pytest
+
+import software.python_bindings as tbots
+from proto.play_pb2 import Play
+from software.simulated_tests.ball_enters_region import *
+from software.simulated_tests.simulated_test_fixture import simulated_test_runner
+from proto.message_translation.tbots_protobuf import create_world_state
+from proto.ssl_gc_common_pb2 import Team
+from proto.ssl_gc_geometry_pb2 import Vector2
+
+# TODO issue  #2599 - Remove Duration parameter from test
+@pytest.mark.parametrize("run_enemy_ai,test_duration", [(False, 10), (True, 20)])
+def test_two_ai_ball_placement(simulated_test_runner, run_enemy_ai, test_duration):
+
+    # starting point must be Point
+    ball_initial_pos = tbots.Point(2, 2)
+    # placement point must be Vector2 to work with game controller
+    ball_final_pos = Vector2(x=-3, y=-2)
```

Comment:
lets use our vector  here and convert to the gamecontroller one in the send_ci_input function to make it a bit easier